lib/omniauth/login_dot_gov/authorization.rb

```diff
@@ -28,12 +29,35 @@ def redirect_url_params
         }
       end
 
+      def aal
+        if client.aal.nil?
+          nil
+        elsif VALID_AAL_VALUES.include?(client.aal.to_s)
+          client.aal
```

Comment:
**question:** do we need to do `client.aal.to_s` to make sure we're getting a string?

Reply:
alternatively we can do string conversion when we initialize the client, that's probably better

Reply:
I think I'd rather do it here as the `to_s` that far up will turn a `nil` into `''`

Reply:
makes perfect sense, nice work!